src/cmd/set.rs

```diff
@@ -146,6 +146,10 @@ impl Set {
         frame.push_bulk(Bytes::from("set".as_bytes()));
         frame.push_bulk(Bytes::from(self.key.into_bytes()));
         frame.push_bulk(self.value);
+        if let Some(ms) = self.expire {
+            frame.push_bulk(Bytes::from("px".as_bytes()));
+            frame.push_int(ms.as_millis() as u64);
```

Comment:
I would like a comment that explains what `px` means.

Reply:
It's expiration in milliseconds, in Redis protocol there are three options:

SET key value
SET key value EX secs
SET key value PX msecs